schedule.md

```diff
@@ -2,71 +2,98 @@
 layout: default
 title: "Schedule"
 author: "Community"
-image: mountains.jpg
 ---
 
-The workshop will take place from **Monday 13** till **Friday 17 July 2020**.
+The workshop will take place from **Monday 13th** till **Friday 17th July 2020**.
 
-All times mentioned below are CEST (UTC+2:00).
+All times mentioned below are in **central European time** (CEST, UTC+2:00).
+Most of the schedule below form the *beginner track* of the workshop.
+Intermediate users can work on pair programming with more advanced
+users at any time, using the 
+[dedicated Zulip stream](https://leanprover.zulipchat.com/#narrow/stream/238830-Lean-for.20the.20curious.20mathematician.202020)
+for coordination.
+
+Each session will start with a short presentation that either will have
+been pre-recorded or will be live-recorded and available here during and
+after the session.
+Then it will move to an exercises session using online chat and
+screen-sharing.
 
 ### Monday:
 
+The goal of this first day is to make sure that all participants have
+Lean and its supporting tools installed, and to start proving things
+without knowing anything about foundations or theory building,
+using the [Natural Number Game](http://wwwf.imperial.ac.uk/~buzzard/xena/natural_number_game/).
+
+In parallel during the afternoon, intermediate users will have the
+opportunity to learn about Lean meta-programming framework, which allows
+to automate some proofs.
+
 Time | Activity | Speaker
 ---- | -------- | -------
-07:00 -- 13:00 | Tech support (×) |
-13:00 -- 13:30 | Welcome + 1st Lean proof | Scott Morrison
-13:30 -- 14:00 | Natural number game demo | Kevin Buzzard
-14:00 -- 15:00 | Natural number game “exercise session” |
-15:00 -- 17:00 | Exercises and Q&A on Zulip |
+07:00 -- 12:00 | Tech support (installing Lean, using git + GitHub |
+13:00 -- 13:30 | Welcome + 1st Lean proof                 | Scott Morrison
+13:30 -- 18:00 | Natural number game (demo and exercises) | Kevin Buzzard
+13:30 -- 18:00 | Meta-programming and tactic writing      | Rob Lewis
```

Comment:
Whoa, I'm bumped from 1 hour to 4.5?

Reply:
This seems very very early to teach tactic writing. I guess this isn't part of the "beginner stream"?

Reply:
Nope, it isn't, see the text above this table.